### ISSUE TITLE
[webui] Allow timeline scroll-to-zoom without holding ALT

### ISSUE THREAD

**Issue by ericl** [ray-project/ray#979]
This can be done by patching catapult as follows:

```
--- a/tracing/tracing/ui/timeline_track_view.html
+++ b/tracing/tracing/ui/timeline_track_view.html
@@ -577,7 +577,7 @@ Polymer({
   },
 
   onMouseWheel_(e) {
-    if (!e.altKey) return;
+    if (e.altKey) return;
 
     const delta = e.wheelDelta / 120;
     const zoomScale = Math.pow(1.5, delta);
```

cc @robertnishihara @alanamarzoev @Wapaul1

**Comment by robertnishihara** [comment]
Let's do that. We can fork it and just modify 

https://github.com/ray-project/ray/blob/1eb8c833141b919aa4e60918dfecbace03a4fe68/src/thirdparty/build_ui.sh#L35

to pull the forked version.

We could submit a PR to catapult, but given the discussion it seems very unlikely that this change would get accepted upstream.

**Comment by ericl** [comment]
Could you for catapult in the ray-project org with the patch? I don't have permissions.

**Comment by robertnishihara** [comment]
Fixed by #993.
